_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: default
+title: Index management
+parent: Admin UI for index operations in OpenSearch Dashboards
+nav_order: 10
+---
+
+# Index management
+Introduced 2.5
+{: .label .label-purple }
+
+The Index Management section in the admin UI allows you to perform the operations available in the [Index API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/index/) from OpenSearch Dashboards' web UI.
+
+## Index policies
+
+[Policies]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/) are configurations that define the possible states of an index, the actions to perform when an index enters a state, and the conditions that must be met to transition between states:
+
+1. **States**: The possible states of an index, including the default state for new indexes. For example, you might name your states `hot`, `warm`, or `delete`. For more information, see [States]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#states).
+2. **Actions**: Any actions that you want the plugin to take when an index enters a state, such as performing a rollover. For more information, see [Actions]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#actions).
+3. **Transitions**: The conditions that must be met for an index to move into a new state. For example, if an index is more than eight weeks old, you might want to move it to the `delete` state. For more information, see [Transitions]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#transitions).
+
+You can also upload a JSON document to specify an index policy.
+{: .note}
+
+You have complete flexibility in the way you can design your policies. You can create any state, transition to any other state, and specify any number of actions in each state.
+
+To attach policies to indices, perform the following steps:
+
+1. Under **Index Management**, select **Index policies**.
```

Comment:
Change "select" to "choose". Global:

- Use _choose_ to describe moving to a UI component such as a tab or pane or taking action on a button or menu.
- Use _select_ to describe picking a user-specific resource or enabling one of several options. Contrast with _clear_ to turn off previously selected options.